interfaces/builtin/nvidia_tegra_drivers_support.go

```diff
@@ -0,0 +1,90 @@
+// -*- Mode: Go; indent-tabs-mode: t -*-
+
+/*
+ * Copyright (C) 2024 Canonical Ltd
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License version 3 as
+ * published by the Free Software Foundation.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ *
+ */
+
+package builtin
+
+const nvidiaTegraDriversSupportSummary = `allows iGPU access to NVIDIA Tegra platforms`
+
+const nvidiaTegraDriversSupportBaseDeclarationSlots = `
+  nvidia-tegra-drivers-support:
+    allow-installation:
+      slot-snap-type:
+        - core
+    deny-auto-connection: true
+`
+
+const nvidiaTegraDriversSupportConnectedPlugAppArmor = `
+@{PROC}/sys/vm/mmap_min_addr r,
+
+# nvidia-smi is trying to access /sys/bus/nvmem/devices/fuse/nvmem
+# but doesn't have permissions to the link location which is listed here
+/sys/devices/platform/bus@*/*.fuse/fuse/nvmem r,
+
+/dev/nvmap rw,
```

Comment:
Can you provide a link to any documentation which describes these interfaces? Also can you please show the output for `ls -l` for each of these files on a system since we should ensure the interface privilege level is appropriate for the device file permissions. Thanks.